assets/show-more.js

```diff
@@ -0,0 +1,22 @@
+class ShowMoreButton extends HTMLElement {
+  constructor() {
+    super();
+    const button = this.querySelector('button');
+    button.addEventListener('click', (event) => {
+      this.expandShowMore(event);
+      const nextElementToFocus = event.target.parentNode.parentNode.parentNode.querySelector(".show-more-item")
```

Comment:
It would be best to not traverse the DOM via chained `parentNode`. This adds fragility and may lead to unexpected results if even a small change in the markup is made.

Reply:
Good point https://github.com/Shopify/dawn/pull/1443/commits/e7fbe88461b3fb873381185f1e38c14f367b6dc6